js/modules/k6/experimental/streams/module.go

```diff
@@ -285,3 +290,49 @@ func (mi *ModuleInstance) NewReadableStreamDefaultReader(call goja.ConstructorCa
 
 	return object
 }
+
+// NewReadableStreamForReader is the equivalent of [NewReadableStreamDefaultReader] but to initialize
+// a new [ReadableStream] from a given [io.Reader] in Go code.
+// It is useful for those situations when a [io.Reader] needs to be surfaced up to the JS runtime.
+func NewReadableStreamForReader(vu modules.VU, reader io.Reader) *goja.Object {
```

Comment:
```suggestion
func NewReadableStreamFromReader(vu modules.VU, reader io.Reader) *goja.Object {
```

Is `For` intentional in this case?

Reply:
Yeah, it was intentional, but I don't have a strong opinion.